tests/Kiota.NPM.IntegrationTests/Assets/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "kiota-consumer",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
+    "@microsoft/kiota": "file:./microsoft-kiota-1.0.0.tgz"
```

Comment:
Can we wildcard this.

Reply:
Or add a command line option in the test to force it to be 1.0.0

Reply:
Hi @garethj-msft, could you share more context on this?